transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Development/Production Mode Tutorial
+permalink: /transformations/dev-prod-mode/
+---
+
+* TOC
+{:toc}
+
+The feature Development/Production Mode enables you to develop code above your production data without affecting
+or endangering the data in any way. It works as a version system where you create a new branch, make changes 
+in it, and run your configurations as needed. Output data is stored separately and does not overwrite your
+production data. 
+
+In this tutorial, we will show you how Development/Production Mode works, what the process of developing in 
+a branch looks like, and what you need to do before you can start.
+
+## Private Beta Warning
+Development/Production Mode is currently in private beta. Some features may not work as expected. Please bear 
+with us while we polish all necessities. Any feedback is welcome at ideas@keboola.com.
+
+## Before You Start
+Before you start using Development/Production Mode, you need to create a testing project under 
+the [Free Plan](/management/payg-project/) (available on our [website](https://www.keboola.com/)). 
+Send your request for testing this beta feature to Keboola Support with the project URL/ID, for example,
```

Comment:
To je divná instrukce - mají normálně poslat request z projektu (my tam to URL uvidíme automaticky).

Reply:
Tohle mi napsala Kristyna :-(

Reply:
j,  ahoj:),  sorry for the drive-by :-).  I would put here: `To request to be a beta tester for this feature please ask via the support button in your project` or something like that 🍻